src/main/java/TaskManager.java

```diff
@@ -0,0 +1,134 @@
+public class TaskManager {
+    private Task[] taskList;
```

Comment:
Plural form should be used on names representing a collection of objects

Reply:
Noted, since this is an array of `Task`s, I will be changing it to `tasks`.